Library/Homebrew/brew.sh

```diff
@@ -55,6 +44,37 @@ else
   HOMEBREW_DEFAULT_TEMP="/tmp"
 fi
 
+realpath() {
+  (cd "$1" &>/dev/null && pwd -P)
+}
+
+# Support systems where HOMEBREW_PREFIX is the default,
+# but a parent directory is a symlink.
+# Example: Fedora Silverblue symlinks /home -> var/home
```

Comment:
@osalbahr @iMichka should this say `/var/home`?

Reply:
I think yes, that's a typo in the comment. @osalbahr up for a follow up PR?

Reply:
Great, thanks!